classes/log/store.php

```diff
@@ -68,6 +78,17 @@ public function __construct(log_manager $manager) {
      *
      */
     protected function is_event_ignored(event_base $event) {
+
+        if ((!CLI_SCRIPT or PHPUNIT_TEST) and !$this->logguests) {
```

Comment:
Use `&&` instead of `and` and use `||` instead of `or`. The two have different precedence and I don't want other contributors to get tripped up by that.